client/one-time-payment/react/src/App.tsx

```diff
@@ -0,0 +1,44 @@
+import { useState, useEffect } from "react";
+import PayPalButton from "./components/PayPalButton";
+import VenmoButton from "./components/VenmoButton";
+
+function App() {
+  const [isSDKReady, setIsSDKReady] = useState<boolean>(false);
```

Comment:
Suggestion: Since we are looking at these values further in the component tree, we should make this a context

```
// sdkContext.ts
export const PayPalSDKContext = React.createContext({
  isReady: false
})
```

Then in our render
```
return (
  <PayPalSDKContext.Provider value={{isReady: isSDKReady}}>
     ...existing logic
  </PayPalSDKContext.Provider>
)
```

Finally, in our Buttons components:
```
const Button = () => {
  const {isReady} = useContext(PayPalSDKContext)

  return isReady ? <paypal-button /> : null
}
```